BaseApp/COD50003.txt

```diff
@@ -0,0 +1,91 @@
+OBJECT Codeunit 50003 DotNet_XslTransform
+{
+  OBJECT-PROPERTIES
+  {
+    Date=;
+    Time=;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      DotNetXslTransform@17024400 : DotNet "'System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Xml.Xsl.XslCompiledTransform";
+
+    [External]
+    PROCEDURE CreateFromBlob@17024400(VAR TempBlob@17024400 : Record 99008535);

```

Comment:
Having such a function makes total sense, however, we should try to isolate helper functions from the actual wrappers. A wrapper codeunit cannot take any dependecy on the base application (e.g. TempBlob), as the BaseApplication eventually will have to take a dependency on the wrapper codeunits. Ideally, this function would go into an XML helper codeunit, much like COD10 Type Helper, but a little more structured ;-)